src/ucm/util/reloc.h

```diff
@@ -39,12 +41,33 @@ ucs_status_t ucm_reloc_modify(ucm_reloc_patch_t* patch);
 /**
  * Get the original implementation of 'symbol', which is not equal to 'replacement'.
  *
+ * This function is set to static to make sure that the symbol search done here
```

Comment:
This function is static to make sure the symbol search is done from the context of the shared object which defines the replacement function. If the replacement function is defined in a loadbale module, the symbols it imports from other libraries may not be available in global scope.